src/config_builder.py

```diff
@@ -148,20 +158,23 @@ def _server(self) -> dict:
 
     @property
     def _storage_config(self) -> dict:
-        # Ref: https://grafana.com/docs/loki/latest/configure/#storage_config
-        return {
+        storage_config = {
             "boltdb_shipper": {
                 "active_index_directory": BOLTDB_DIR,
                 "shared_store": "filesystem",
                 "cache_location": BOLTDB_CACHE_DIR,
             },
-            "tsdb_shipper": {
+            "filesystem": {"directory": CHUNKS_DIR},
+        }
+
+        if self.v12_migration_date:
```

Comment:
Does this need to be rendered conditionally? I.e. does loki care that there's an unused section?

Reply:
Loki itself wouldn't care but it will create some files in this dir that will be redundant with no actual schema